frontend/public/style/_overrides.scss

```diff
@@ -148,6 +148,10 @@ h6 {
   color: var(--pf-c-button--m-plain--Color);
 }
 
+.pf-c-dropdown__toggle.query-browser__span-dropdown-button .pf-c-dropdown__toggle-icon {
```

Comment:
This selector is incredibly specific, so we'll have this same bug any time a similar component combo is used.

This seems like an upstream bug, no?

Reply:
Well I don't think so, I think the larger issue is we're using a PF4 `<TextInput>` along with our custom `<Dropdown>` that has no text (the default margin is to separate that text from `pf-c-dropdown__toggle-icon`). It seems this should be a PF4 `<Select>`

I could make it reduce specificity some and `!important` to give priority?
```
.query-browser__span-dropdown-button .pf-c-dropdown__toggle-icon {
  margin-left: 6px !important;
}
```